test/cucumber/steps/EvaluationComparison.groovy

```diff
@@ -0,0 +1,115 @@
+package steps
+
+import pages.EvaluationComparisonPage
+import pages.ShowComparisonPage
+import ta.Student
+import ta.StudentController
+
+
```

Comment:
Falta:
this.metaClass.mixin(cucumber.api.groovy.Hooks)
this.metaClass.mixin(cucumber.api.groovy.EN)